modular-docs-manual/content/topics/module_reusing-modules-procedure.adoc

```diff
@@ -17,7 +17,7 @@ This error is resolved by adding and defining a document variable.
 [discrete]
 .Procedure
 
-. In the module file that will be reused, add the `+++{context}+++` suffix with an underscore to the anchor name in the format `[id='anchor-name_+++{context}'+++]`.
+. In the module file that will be reused, add the `+++{context}+++` suffix with a dash to the anchor name in the format `[id='anchor-name-+++{context}'+++]`.
```

Comment:
One instance of 'dash' instead of 'hyphen'.

Reply:
Fixed! Thank you.